homeassistant/components/vlc_telnet/media_player.py

```diff
@@ -24,6 +24,7 @@
     SUPPORT_STOP,
     SUPPORT_VOLUME_MUTE,
     SUPPORT_VOLUME_SET,
+    SUPPORT_VOLUME_STEP,
```

Comment:
We don't need to add this support since we already support volume set. The volume up/down services should work already.

Reply:
Is there something else I need to set to enable it? (and/or could there be a bug in the base media class?)

<img src="https://user-images.githubusercontent.com/637285/109440337-f50fb200-79ff-11eb-8721-912ffdeb7e02.png" height=300> <img src="https://user-images.githubusercontent.com/637285/109584804-44251800-7ad0-11eb-8a4e-c913c4966a24.png" height=300>

(left: without `SUPPORT_VOLUME_STEP`, right: with `SUPPORT_VOLUME_STEP`)

Reply:
Ah, perhaps this is the issue: https://github.com/home-assistant/core/blob/98be703d90e44efe43b1a17c7e5243e5097b00b1/homeassistant/components/media_player/__init__.py#L231-L236

Maybe line 235 should be `[SUPPORT_VOLUME_SET | SUPPORT_VOLUME_STEP]`?

Reply:
It could be that the frontend uses a different check than the backend. The backend only requires any of those features in the list to be supported.

@bramkragten do you know if the frontend requires `SUPPORT_VOLUME_STEP` to show the volume +/- buttons?

Reply:
@dmcc you can test the service directly in the dev tools services panel.

Reply:
Yes, it does atm. But I think, we kinda assume you rather have a slider than buttons, and both is a bit overkill...?

Reply:
@dmcc do you specifically want to have the volume +/- buttons in the frontend, or is it enough to be able to call the service in eg an automation?

Reply:
```suggestion
```

Reply:
I've confirmed that services will work even if `SUPPORT_VOLUME_STEP` isn't present.

I do sometimes use the +/- buttons on mobile where the slider is harder to use, but it's not a big problem. I can always make buttons that call the services, which might be a better UX anyway, so I'm changing this PR to just `SUPPORT_VLC` elements.